CONTRIBUTING.md

```diff
@@ -0,0 +1,333 @@
+* Tag `@team-deved` for review.
+* Once approved and merged into `main`, content is automatically updated on `/docs`.
+
+### CI
+
+CI runs in [CircleCI](https://circleci.com/).
+
+### Reviews
+
+Request reviews from the group `@team-deved` on any pull requests.
+
+[CodeRabbitAI](https://github.com/apps/coderabbitai) doesn't run by default on pull requests, but is available using the command `@coderabbitai review`.
+
+### Linting
+
+Contributors should use [EditorConfig](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) when adding content. An `.editorconfig` file is included in the repository.
+
+### Style guide
+
+There is a [writing style guide](writing-style-guide.md) that contributors should follow when adding content.
+
+## Repository structure
+
+There are two main locations that contributors need to work with to varying extents. The following is a table explaining what each folder contains that is relevant to editing the docs:
+
+| Folder | Contains |
+| ------ | -------- |
+| content | All content pages written in textile. |
+| src | Images, navigation and language version and management. |
+
+## Textile format
+
+The following sections discuss how to write and implement each component using textile.
+
+### Metadata
+
+The following metadata can be included in the front matter of files. Metadata should be written between `---` in YAML:
+
+```yaml
+---
+title: "The page title that will appear on page. This is required."
+meta_description: "A description for the page that will be returned in searches. This is required."
+meta_keywords: "A comma separated list of keywords about the page."
+redirect_from: "A YAML array of pages that will redirect to the current page."
+---
+```
+
+#### Redirects
+
+Redirects are automatically added to [`config/nginx-redirects.conf`](config/nginx-redirects.conf) using the values set in `redirect_from`, and used to create a map of nginx redirects at build time.
+
+Other one-off instances of redirects are contained in the following files. These should rarely, if ever need to be touched.
+
+* [`/config/website-redirects.conf`](config/website-redirects.conf)
+* [`/config/client-lib-development-guide-redirects.conf`](config/client-lib-development-guide-redirects.conf)
+* [`/config/nginx.conf.erb`](config/nginx.conf.erb)
+
+### Headings
+
+Page headings are implemented in the format: `h[1-6](#anchor-id).`. In reality `h1` is never used as it is automatically set by the page title defined in the metadata.
+
+An example heading (with a custom anchor link) is: `h2(#subscribe). Subscribe to a channel`
+
+### Links
+
+Links in textile are written in quotation marks. Link text can also be [code styled](#in-line-code).
+
+* Link to another heading on the same page: `"channel state changes":#listen-for-state`
+* Link to another docs page: `"messages":/docs/channels/message`
+* Link to an external page: `"Ably dashboard":https://ably.com/dashboard`
```

Comment:
Maybe separate this one a little more? With sub-titles?
Internal links and external links? 

I only say this as new contributors often spend the most time bug-fixing links that won't work :)